src/bsp/src/stm32/src/interloc/src/InterlocManager.cpp

```diff
@@ -41,8 +41,8 @@ void InterlocManager::startInterloc() {
     syncClocks();
 
     // Uncomment one of the following lines to go into TWR
-    // m_stateHandler.setState(InterlocStates::WAIT_POLL);
-    // m_stateHandler.setState(InterlocStates::SEND_POLL);
+    // m_stateHandler.setState(InterlocStates::WAIT_POLL, InterlocEvent::NO_EVENT);
+    // m_stateHandler.setState(InterlocStates::SEND_POLL, InterlocEvent::NO_EVENT);
```

Comment:
Commented code?

Reply:
Yes, this starts the interloc in one of two modes, but won't work if no hivesight is on the board so it is just there to make it easier when testing without having to always rewrite these lines